src/core/sensor/qgsiodevicesensor.h

```diff
@@ -279,6 +290,8 @@ class CORE_EXPORT QgsSerialPortSensor : public QgsIODeviceSensor
 
     QString mPortName;
 
+    QString mBaudrate;
```

Comment:
Let's make it the right type to begin with, we don't need to set ti as a string member.

```suggestion
    QSerialPort::BaudRate mBaudrate;
```